pkg/glusterfs/dockerfile/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Gluster CSI Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM centos:7.4.1708
+
+# Copy glusterfsplugin from build _output directory
+COPY glusterfsplugin /glusterfsplugin
+
+RUN yum -y install glusterfs-fuse && yum -y install epel-release && yum -y install jq && yum clean all
```

Comment:
Which Gluster FUSE client version is this?

Reply:
afaik,  we dont have to worry about gluster fuse version in this dockerfile.